galley/pkg/config/analysis/analyzers/virtualservice/destinationhosts.go

```diff
@@ -134,10 +135,29 @@ func initServiceEntryHostMap(ctx analysis.Context) map[util.ScopedFqdn]*v1alpha3
 		}
 		return true
 	}
+	extractSvc := func(r *resource.Instance) bool {
```

Comment:
It would be helpful to document the intent of this code to overload ServiceEntry to represent k8s services as well.